tools/scxtop/src/main.rs

```diff
@@ -158,6 +162,17 @@ async fn main() -> Result<()> {
                 tx.send(action).ok();
             }
             #[allow(non_upper_case_globals)]
+            event_type_SOFTIRQ => unsafe {
```

Comment:
nit: this `unsafe` block is bigger than it needs to be. Could we settle on something like:

```
let softirq_event = unsafe { &event.event.softirq };
```

So we don't accidentally use other unsafe things in this block?

Reply:
I think anything accessing the union has to be unsafe so the only thing that isn't unsafe is `cpu`. I'll maybe live lazy this time 🗡️